public/pages/Snapshots/containers/Snapshots/Snapshots.tsx

```diff
@@ -240,20 +243,51 @@ export default class Snapshots extends Component<SnapshotsProps, SnapshotsState>
     this.setState({ showRestoreFlyout: false });
   };
 
+  onClickTab = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    const { selectedItems } = this.state;
+    const target = e.currentTarget;
+    const snapshotPanel = target.textContent === "Snapshots" ? true : false;
+    const prev = target.previousElementSibling;
+    const next = target.nextElementSibling;
+
+    if (selectedItems.length === 0) {
+      this.context.notifications.toasts.addWarning("Please select a snapshot to view restore activities");
+      return;
+    }
+
+    this.context.chrome.setBreadcrumbs([BREADCRUMBS.SNAPSHOT_MANAGEMENT, BREADCRUMBS.SNAPSHOTS]);
+
+    target.ariaSelected = "true";
+    target.classList.add("euiTab-isSelected");
+
+    if (prev) {
+      prev.classList.remove("euiTab-isSelected");
+      prev.ariaSelected = "false";
+    }
+    if (next) {
+      next.classList.remove("euiTab-isSelected");
+      next.ariaSelected = "false";
+    }
```

Comment:
Not sure what this part does, can you give some explain

Reply:
this is dealing with navigating to one panel or another when a user clicks on a tab.  it makes sure the tab clicked becomes the selected tab and that the other one is set to not be the selected one.  eiuTab-isSelected is what gives it the animation/blue highlighting.